includes/class-module.php

```diff
@@ -116,9 +116,37 @@ public function load_args_from_layout_content( $content ) {
 
 	/**
 	 * Render module template.
+	 *
+	 * @param string  $raw_content Raw ACF layout content.
+	 * @param boolean $echo Echo content.
 	 */
-	public function render_template() {
+	public function render_template( $raw_content, $echo = true ) {
+
+		// Load module data from raw ACF layout content.
+		$this->load_args_from_layout_content( $raw_content );
+
+		if ( false === $echo ) {
+			ob_start();
+		}
+
+		// Global HTML wrapper tag.
+		$wrapper_tag = apply_filters( 'hogan/module/wrapper_tag', 'section' );
+
+		// Override wrapper tag for module.
+		$wrapper_tag = apply_filters( 'hogan/module/' . $this->name . '/wrapper_tag', $wrapper_tag );
+
```

Comment:
Kunne kanskje vært greit med en action hook før og etter template så man kan gjøre noe her.
Eks. `do_action('hogan/module/' . $this->name . '/template/pre');`

Reply:
Dette er nå lagt til.